includes/snmp.inc.php

```diff
@@ -264,6 +282,29 @@ function snmp_get($device, $oid, $options = null, $mib = null, $mibdir = null)
 }//end snmp_get()
 
 
+function snmp_get_next($device, $oid, $options = null, $mib = null, $mibdir = null)
```

Comment:
mib polling afaik should just walk through genEquipRadioStatusDefectedBlocks and pick out the data it finds. Have you tried just specifying that?